code/modules/antagonists/abductor/equipment/gear/abductor_clothing.dm

```diff
@@ -103,6 +103,8 @@
 
 /obj/item/clothing/suit/armor/abductor/vest/proc/return_disguise_name(mob/living/carbon/human/source, list/identity)
 	SIGNAL_HANDLER
+	if(identity[VISIBLE_NAME_FORCED]) // name-forcing overrides disguise
+		return
```

Comment:
This is entirely at the whim of the order the signals are registered, which stinks and is a bad pattern

Reply:
I addressed this particular array in all the places I found that it's manipulated. Did I miss some or is there a flaw in my logic? I did several rounds of testing.